crates/samples/direct2d/src/main.rs

```diff
@@ -209,15 +209,15 @@ impl Window {
         unsafe {
             target.SetTransform(&(Matrix3x2::rotation(angles.second, 0.0, 0.0) * translation));
 
-            target.DrawLine(D2D_POINT_2F::default(), D2D_POINT_2F { x: 0.0, y: -(radius * 0.75) }, brush, radius / 25.0, &self.style);
+            target.DrawLine(&D2D_POINT_2F::default(), &D2D_POINT_2F { x: 0.0, y: -(radius * 0.75) }, brush, radius / 25.0, &self.style);
```

Comment:
These should be passed by value (when the underlying metadata passes structs by value). It's a fine line because you don't want to pass non-blittable structs by value even if the underlying metadata does.

Reply:
`type_def_is_blittable` or `type_is_blittable` should work.

Reply:
Done!